.github/workflows/ci.yaml

```diff
@@ -47,6 +47,8 @@ jobs:
       run: bin/setup
     - name: Run tests
       run: bundle exec rake
+    - name: Run core security tests
+      run: bundle exec rake spec:security
```

Comment:
Note, this repo doesn't have suites so I added this step to run after running the normal tests.